httpcallback/api_dereg_notify.go

```diff
@@ -50,7 +50,8 @@ func HTTPDeregistrationNotification(c *gin.Context) {
 	if supi, exists := c.Params.Get("supi"); exists {
 		req.Params["supi"] = supi
 	}
-	rsp := producer.HandleDeregistrationNotification(req)
+	ctx := c.Request.Context()
+	rsp := producer.HandleDeregistrationNotification(ctx, req)
```

Comment:
Given that `ctx` is only used once, why not just as follows?
```suggestion
	rsp := producer.HandleDeregistrationNotification(c.Request.Context(), req)
```

Reply:
sure, done